pytorch_blade/src/BUILD

```diff
@@ -106,6 +106,7 @@ pybind_library(
     ] + if_tensorrt_enabled([
         "@tensorrt//:nvinfer_headers",
     ]),
+    alwayslink = 1,
```

Comment:
Is the alwayslink must be set to true.

Reply:
Yes, since this pybind library is not directly referenced by any C++ code, it will be dropped when linking if the default whole-archive behavior is disabled.